scripts/get-files-touched-by.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." >/dev/null 2>&1 && pwd )"
+marker=$(mktemp)
+touch $marker -r ${BASH_SOURCE[0]}
+
+CMD=$1
+shift
+
+$CMD "$@" 1>&2
+
+# Find all the files paths that are generated out of the yarn schemas script
```

Comment:
This file doesn't really have anything to do with `yarn schemas`, although only used for it now. What about `# Find all files that were generated by running the command in argv`?

Reply:
You are right I took a shortcut since the script is only used along with yarn schemas but it could be used for other scripts. I will change that, thanks